### PR TITLE
Air quality: cleanup widget location display

### PR DESCRIPTION
**Pull request by f-laurens**
Change  location display in Air quality widget, at the expense of a little screen estate, to be similar to Weather widget.
(Note: this PR does **not** correct issue #245.)

### CODE REVIEW

File: nabairqualityd/aqicn.py [hunk 1]
```diff
@@ -35,15 +38,17 @@ def _aqicn_url(self, lat, lon):
...
             return (
-                "https://api.waqi.info/feed/geo:" + lat + ";" + lon + "/"
-                "?token=4cf7f445134f3fb69a4c3f0e5001e507a6cc386f"
+                AQICN_FEED
+                + "geo:"
+                + lat
+                + ";"
+                + lon
+                + "/?token="
+                + AQICN_TOKEN
```
Comment (mehdichaouch) [inline nabairqualityd/aqicn.py:47]: I'm not a python expert, but using string substitution is maybe better:
`return f"{AQICN_FEED}geo:{lat};{lon}/?token={AQICN_TOKEN}"

Comment (mehdichaouch) [reply nabairqualityd/aqicn.py:47]: Any tips @pguyot?

Comment (pguyot) [reply nabairqualityd/aqicn.py:47]: Indeed f-strings are considered a better practice for Python 3.6+ unless format string is user-supplied.

Comment (f-laurens) [reply nabairqualityd/aqicn.py:47]: Hmmm.... Existing Pynab code makes extensive use of string catenation. I didn't want to disrupt this.
I suggest you conduct an extensive code review/cleanup for 1.0.0.

**Comment by omevel** [comment]
Here are the Air quality animations.
[Nabaztag_Air_Traffic.zip](https://github.com/nabaztag2018/pynab/files/6568483/Nabaztag_Air_Traffic.zip)

**Comment by f-laurens** [comment]
> Here are the Air quality animations.

Thanks. This applies to PR #250.

**Comment by f-laurens** [comment]
Set to draft wihile rebasing...

**Comment by f-laurens** [comment]
Ready for merge...

**Pull request merged**